### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by arush-sal**
`26452620` - Add lifecycle documentation.
`eab8ee54` - Update Readme
`f447c218` - Update AWS documentation

### CODE REVIEW

File: docs/en/provisioner/aws/lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+Once executed a confirmation would be needed to overwrite the existing inventory file post which above command will scale your infrastructure as well as the Kubernetes cluster together to the specified capacity.
+
+## Reset the cluster
+
+Make sure you are in the same directory where you executed `tk8 cluster install aws` with the inventory directory.
+If you use a different workspace name with the --name flag please provide it on resetting too.
+
+To scale the provisioned cluster run:
```
Comment (MuellerMH) [inline docs/en/provisioner/aws/lifecycle.md]: to reset the ...

**Review by MuellerMH — CHANGES_REQUESTED** [review]
lgfm with one change :) 👍

File: docs/en/provisioner/aws/lifecycle.md (hunk 1 above)
Comment (arush-sal) [reply docs/en/provisioner/aws/lifecycle.md]: Oops! Sorry about that :(

**Review by MuellerMH — APPROVED** [review]

File: docs/en/provisioner/aws/lifecycle.md (hunk 1 above)
Comment (arush-sal) [reply docs/en/provisioner/aws/lifecycle.md]: @MuellerMH Fixed it in 28a2a80

**Comment by MuellerMH** [comment]
lgfm 
Thank you so much for all the corectures.

**Review by arashkaffamanesh — COMMENTED** [review]
fine

**Pull request merged**